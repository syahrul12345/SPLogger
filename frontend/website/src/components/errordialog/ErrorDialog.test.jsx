import React from "react";
import renderer from "react-test-renderer";
import { shallow } from "enzyme";

import ErrorDialog from "./ErrorDialog";

describe("Render tests for error dialog", () => {
  test("Comapre snapsot", () => {
    const open = false;
    const setOpen = jest.fn();
    const component = renderer.create(
      <ErrorDialog open={open} setOpen={setOpen} errorMessage={"Test error"} />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
  test("Check for error message", () => {
    const open = false;
    const setOpen = jest.fn();
    const component = shallow(
      <ErrorDialog open={open} setOpen={setOpen} errorMessage={"Test error"} />
    );
    expect(component.find("#alert-dialog-description").text()).toEqual(
      "Test error"
    );
  });
});
