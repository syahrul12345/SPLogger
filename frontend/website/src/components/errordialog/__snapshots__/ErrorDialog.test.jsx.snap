// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render tests for error dialog Comapre snapsot 1`] = `<div />`;
